File: pert4/sample.py
class Nama:
        def full(self, firt_name, middle_name, last_name):
                print("FULL NAME :", first_name, middle_name, last_name
    
def main():
n = Nama()

while True:
    print("MENU SOAL 4 : ")
    PRINT("1. input")
    print("2. keluar\n")

    pilih = input("Pilihan anda : ")

    if pilih == '1':
        first_name = input ("Masukan Nama Depan : ")
        midle_name = input ("Masukan Nama Tengah : ")
        last_name = input ("Masukan Nama Belakang : ")
        n.full(first_name, middle_name, last_name )
    elif pilih == '2':
        print("\nTERIMA KASIH")
        break
    else:
        print("pilihan invalid")

        pil = input("Ingin kembali ke menu ? (YY/N) : ")
        if pil.lower() != 'y':
            print("\nTERIMAKASIH")
            break
            
    if __name__== "_main_":
        main()

File: pert4/sample.cpp
# include <iostream>
#include <string>
using namespace std;

class Nama {
    public :
        string fName, mName, lName;
        void full(string fName, string mName, string lName)
        {
            cout << "FULL NAME : " << fName + " "<< mName << " " << lName << endl;

        }
};

int main() {
    Nama n;
    int pilih;
    char pil; 
menu:
    cout << "MENU" << endl;
    cout << "1. input " << endl;
    cout << "2. Exit " << endl;
    cout << "pilihan : " ;
    cin >> pilih;

    switch(pilih){
        case 1;
        cout << "Masukan Nama Depan : " ;
        cin >> n.fName;
        cout << "Masukan Nama Tengah : " ;
        cin >> n.mName;
        cout << "Masukan Nama Akhir : " ;
        cin >> n.lName;
        n.full(n.fName, n.mName, n.lName);
        break;
    case 2:
        goto keluar;
        break;

    default:
        cout << "Pilihan Salah" << endl;
        break;
    }

    cout << "Ingin Mengulangi ? (Y/N) :";
    cin>> pil;
    if ((pil == 'Y' || pil == 'y'))
       {goto menu;}
    else if ((pil == 'N' || pil == 'n'))
       {goto keluar;}

    keluar:
        cout << "\n TERIMA KASIH" << endl;
        
        return 0;
}